packages/opal-server/opal_server/config.py

```diff
@@ -231,6 +231,8 @@ class OpalServerConfig(Confi):
         description="Server is leader or follower",
     )
 
+    SCOPES = confi.bool("SCOPES", default=False)
+
     REDIS_URL = confi.str("REDIS_URL", default="redis://localhost")
```

Comment:
We should have only 1 URL - `BROADCASTER_URL` (maybe rename it to `BACKEND_DB_URL`), which would be used for both features. 
If `SCOPES=on` check that URL begins with "redis://" and print a descriptive error message otherwise.

That way it would be clear to both users and future us that those should be the same. (And if needed, make `scopes` support more platforms, or limit `broadcaster` to redis support only)

Reply:
I think we should keep it as is, and then next week do it together with completely disabling broadcaster backends other than Redis (need to check if any current users use PgSQL/Kafka)